### PR TITLE
MAUI-2854 UG documentation for logarithmic axis in .NET MAUI chart

### PR DESCRIPTION
**Pull request by RajaRamalingam3772**
### Feature description ###

UG documentation for the cartesian chart logarithmic axis feature.

### Analysis and design ###

The logarithmic axis' content is presented in a way that explains the feature in detail.

### Solution description ###

To help the user understand the axis, axis range, and axis base, content and appropriate images have been added to the documentation.

### Output screenshots ###

Logarithmic axis:

maui_chart_logarithmic_axis

Logarithmic axis range customization:

maui_chart_logarithmic_axis_range

Logarithmic axis base customization:

maui_chart_logarithmic_axis_base

Logarithmic interval:

Logarithmic axis interval

Actual minimum & actual maximum:

maui_chart_logarithmic_axis_actual-minimum-actual-maximum

### Areas affected and ensured ###

None

### API Changes ###

None

### Behavioral Changes ###

None

### Test cases ###

None

### Testbed sample location ###

None

### Does it have any known issues?

Not applicable

### Does it have memory leak?

Not applicable

### MR CheckList ###

- [x] Does it follow the design [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? It is mandatory that, we should not move the changes without reading this.
- [ ] Did UI automation passed without errors? If it is customer issue, make sure it is included in the IR automation. Not applicable.
- [ ] Properly working in Xamarin.Forms [previewer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer?tabs=vswin). Not applicable
- [ ] Ensured in iOS, Android, UWP and macOS (if supported). Not applicable
- [ ] Have you added API comments if it is new API? Not applicable
- [ ] Does it need localization? If so, update resx file in [GitHub](https://github.com/syncfusion/xamarin-localized-texts). Not applicable
- [ ] Whether we have added new APIs? If so, whether it is as per the [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? Not applicable
- [ ] If changes are made in the Xamarin.Forms sample browser project file, whether it is included in the props file too? If not, your sample level changes will not reflect in the application published in the hockey app. Not applicable
- [ ] Have you ensured the changes in Android API 19 and iOS 9? Not applicable

### CODE REVIEW

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 1]
```diff
@@ -334,6 +335,135 @@ chart.XAxes.Add(primaryAxis);
...
+
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis range customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_range.jpg)
+
+## Customizing the logarithmic base
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:415]: @RajaRamalingam3772 , why its in H2 font?

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 2]
```diff
@@ -334,6 +335,135 @@ chart.XAxes.Add(primaryAxis);
...
+
+The log base can be customised by using the [LogarithmicBase](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.LogarithmicAxis.html#Syncfusion_Maui_Charts_LogarithmicAxis_LogarithmicBase) property of the axis. By default, the base of a logarithmic axis is set to 10. You can, however, set the base to any value that meets your requirements. Just ensure that the base value is any positive number greater than 1. The following code example demonstrates the logarithmic values in the y axis calculated from base 2.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:417]: The hyper link added to the API was wrong, who is instructed to add like this?

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 3]
```diff
@@ -334,6 +335,135 @@ chart.XAxes.Add(primaryAxis);
...
+
+chart.YAxes = new LogarithmicAxis() { LogarithmicBase = 2 };
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis base customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_base.jpg)
+
+## Logarithmic Interval
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:441]: Sub titles was changing 
`Logarithmic Interval`
`Customizing the logarithmic base`

Follow any of the one pattern, check with numeric axis name the tile like that.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 1 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Now I changed it to H3 font.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 2 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , The link was added by me mistakenly. Now I removed the link.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 3 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Now I followed the pattern link numerical axis in the titles.

**Comment by RajaRamalingam3772** [comment]
@Saravanan-Madhesh , Requested changes comments were addressed.

File: MAUI/Cartesian-Charts/Axis/Overview.md [hunk 4]
```diff
@@ -16,6 +16,7 @@ Vertical(Y) axis always uses numerical scale. Horizontal(X) axis supports the fo
 * Category
 * Numeric
 * Date time
+* Logarithmic
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Overview.md:19]: Change to logarithm

Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Overview.md]: @Saravanan-Madhesh , Requested change is now committed.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 5]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+. . .
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:364]: Remove unwanted lines.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 6]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+LogarithmicAxis secondaryAxis = new LogarithmicAxis();
+chart.YAxes.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis support in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis.jpg)
+
+### Customizing the range
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:377]: Range customization

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 7]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis range customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_range.jpg)
+
+### Logarithmic base
+
+The log axis base can be customized using the [LogarithmicBase]() property of the axis. By default, the base of a logarithmic axis is set to 10. You can, however, set the base to any value that meets your requirements. Just ensure that the base value is any positive number greater than 1 and cannot be a decimal number. The following code example demonstrates the logarithmic values in the y-axis calculated from base 2.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md]: log axis => logarithmic axis

Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md]: The logarithmic axis base can be customized using the [LogarithmicBase]() property of the axis. By default, the base value is 10. The following code example demonstrates how to customize the base value. 

The above content is enough for base.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 8]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis base customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_base.jpg)
+
+### Logarithmic interval
+
+The logarithmic interval can be customized using the [Interval]() property of the axis. In the logarithmic axis, the interval is based on powers. When the logarithmic base is 10 and the logarithmic interval is 1, then the axis labels are placed at an interval of 10. i.e., 1, 10, 100, 1000, etc. When the logarithmic interval is 2, then the axis labels are placed at an interval of 10^2. i.e., 1, 100, 1000, etc. The default value of the interval is 1, and it cannot be a decimal number.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md]: Axis interval can be customized by using the [Interval]() property. By default, interval will be calculated based on the minimum and maximum value of the provided data. And the default value of the interval is 1. The following code example demonstrates the interval customization of the axis.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 9]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+
+chart.YAxes.Add(new LogarithmicAxis( ) { Interval = 2 });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis interval in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_interval.jpg)
+
+### Actual minimum & Actual maximum
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:461]: Actual minimum & maximum

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 10]
```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
...
+
+The actual minimum property can be used to get the double value that represents the minimum value of the chart axis range and the actual maximum property can be used to get the double value that represents the maximum value of the chart axis range. The following code example demonstrates how to get the actual minimum and actual maximum values of the axis range using the labels.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:463]: The ActualMinimum and ActualMaximum of the LogarithmicAxis used to get actual range of the axis. The following code example demonstrates how to get the actual minimum and maximum.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 5 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Lines removed.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 6 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh ,Title has been changed.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 7 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Content modified as required.

Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Content modified as requested.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 8 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Content modified as requested.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 9 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh , Content modified as requested.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 10 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: @Saravanan-Madhesh ,Content modified as requested.

**Comment by RajaRamalingam3772** [comment]
@Saravanan-Madhesh , Requested changes are updated.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 11]
```diff
@@ -334,6 +335,182 @@ chart.XAxes.Add(primaryAxis);
 
 ![DateTimeAxis range customization support in MAUI Chart](Axis_Images/maui_chart_datetime_axis_range.jpg)
 
+## Logarithmic Axis
+
+The [LogarithmicAxis]() uses a logarithmic scale, and it is very useful in visualising data when the given data range has a big difference. It can be used either on the x-axis or the chart's y-axis. The logarithmic axis cannot be plotted with zero or negative values. The following code example demonstrates how to set the logarithmic axis in the chart.
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:341]: The logarithmic axis cannot be plotted with zero or negative values. 

@RajaRamalingam3772 , remove above line.

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 12]
```diff
@@ -334,6 +335,182 @@ chart.XAxes.Add(primaryAxis);
...
+	Maximum = 10000 
+});
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis range customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_range.jpg)
+
+### Logarithmic base
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:405]: Base customization

File: MAUI/Cartesian-Charts/Axis/Types.md [hunk 13]
```diff
@@ -334,6 +335,182 @@ chart.XAxes.Add(primaryAxis);
...
+
+chart.YAxes.Add(new LogarithmicAxis( ) { LogarithmicBase = 2 });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis base customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_base.jpg)
+
+### Logarithmic interval
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/Types.md:431]: Interval customization

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 11 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md:341]: Changes updated.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 13 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: Changes updated.

File: MAUI/Cartesian-Charts/Axis/Types.md (hunk 12 above)
Comment (RajaRamalingam3772) [reply MAUI/Cartesian-Charts/Axis/Types.md]: Changes updated.

**Comment by RajaRamalingam3772** [comment]
@Saravanan-Madhesh , the requested changes are updated. Please validate.

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine

**Pull request merged**